src/PlatformSockets.h

```diff
@@ -23,10 +27,12 @@
 #endif
 #define EINTR WSAEINTR
 
+#ifndef __MINGW32__
 #define EWOULDBLOCK WSAEWOULDBLOCK
 #define EINPROGRESS WSAEINPROGRESS
 #define ETIMEDOUT WSAETIMEDOUT
 #define ECONNREFUSED WSAECONNREFUSED
```

Comment:
I don't think this part is correct. I assume these are already defined by `crt/errno.h`, but those aren't the correct definitions. We need the WSA definitions, not the POSIX ones.

Reply:
I see. I guess under this case, we'll need to not importing certain headers under mingw?